__tests__/components/cards/User.test.tsx

```diff
@@ -0,0 +1,63 @@
+import UserCard from "@/components/cards/User";
+import { renderWithRedux } from "../../../__mocks__/renderWithRedux";
+import { mockUser } from "../../../__mocks__/fixtures/user";
+import { fireEvent, screen } from "@testing-library/react";
+import "@testing-library/jest-dom";
+import { useRouter } from "next/router";
+
+jest.mock("next/router", () => ({
+  useRouter: jest.fn(),
+}));
+const teamMembers = [mockUser];
+const handleClick = jest.fn(() => {});
+const mockPush = jest.fn();
+const mockUseRouter = {
+  locale: "en",
+  push: mockPush,
+};
+(useRouter as jest.Mock).mockReturnValue(mockUseRouter);
```

Comment:
I don't think you need this mock as you mocked useRouter on line 8

Reply:
we need to mock the returned value as well